packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -922,8 +941,8 @@ export class EksOptimizedImage implements ec2.IMachineImage {
 
     // set the SSM parameter name
     this.amiParameterName = `/aws/service/eks/optimized-ami/${this.kubernetesVersion}/`
-      + ( this.nodeType === NodeType.STANDARD ? 'amazon-linux-2/' : '' )
```

Comment:
We normally try to avoid aesthetic changes in the same PRs with functional changes to make sure there is a clean "diff".